src/main/java/lutchat/Todo.java

```diff
@@ -0,0 +1,22 @@
+package lutchat;
+
+public class Todo extends Task{
```

Comment:
There is a missing whitespace here, it should be,
```suggestion
public class Todo extends Task {
```